common/scala/src/main/scala/org/apache/openwhisk/core/entity/WhiskActivation.scala

```diff
@@ -82,8 +82,8 @@ case class WhiskActivation(namespace: EntityPath,
       if (end != Instant.EPOCH) {
         Map(
           "end" -> end.toJson,
-          "duration" -> (duration getOrElse (end.toEpochMilli - start.toEpochMilli)).toJson,
-          "statusCode" -> response.statusCode.toJson)
+          WhiskActivation.durationAnnotation -> (duration getOrElse (end.toEpochMilli - start.toEpochMilli)).toJson,
```

Comment:
This change of using existing constants can be avoided. Using string literals is fine here